WordPress/UITestsFoundation/XCTestCase+Utils.swift

```diff
@@ -32,3 +32,11 @@ public extension XCTestCase {
     }
 
 }
+
+public struct UIElementNotFoundError: Error {
+    var message: String
+
+    init(message: String) {
+        self.message = message
+    }
+}
```

Comment:
Nitpick: What do you think of moving this to its own file?

It's a small type and fits in the Utils umbrella, but I feel it would be tidier on its own.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/22738/commits/434fd4f5db92a7ea9b03dda767a17bcc1022715a.